SR2024/2023-12-13-challenge-reminder.md

```diff
@@ -0,0 +1,15 @@
+Hi $TODO_CONTACT_GIVEN_NAME,
+
+Just a reminder that the first challenge submission deadline passed over the weekend. 
+
+If you aren't aware of the challenges, you can find out more about them in the rulebook [here](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges)
```

Comment:
Please can we avoid using "here" as a link text? It's generally regarded as unclear for users (especially those using assistive technologies) as it provides little description of what the link _is_. Instead we should prefer to use text which describes the target as the link.

```suggestion
If you aren't aware of the challenges, you can find out more about them in the [rulebook](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges).
```